src/EditorFeatures/Test/EditAndContinue/Helpers/SymReaderTestHelpers.cs

```diff
@@ -52,7 +52,7 @@ public static ISymUnmanagedReader5 OpenDummySymReader(ImmutableArray<byte> pdbIm
             pdbImage.WriteToStream(pdbStream);
 
             var pdbStreamCom = SymUnmanagedStreamFactory.CreateStream(pdbStream);
-            if (pdbImage.Length > 4 && pdbImage[0] == 'B' && pdbImage[1] == 'S' && pdbImage[2] == 'J' && pdbImage[3] == 'B')
+            if (pdbImage is [(byte)'B', (byte)'S', (byte)'J', (byte)'B', ..])
```

Comment:
Alternatively, in a battle of the new language features...
```suggestion
            if (pdbImage.Length > 4 && pdbImage[0..4] == "BSJB"u8)
```

Reply:
ooh.  i would do that... but i don't think == works here.  u8 strings are just `ROS<byte>`s, and those don't use `==` to mean value-equality, but reference-equality instead.